src/components/admonition/admonition.types.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * The type of the list, omit to get a regular list. Available types are:
+ * `note`: a grayish note.
+ * `tip`: a greenish tip.
+ * `info`: a blueish info.
+ * `warning`: a yellowish warning.
+ * `danger`: a redish danger.
```

Comment:
```suggestion
 * `danger`: a reddish danger.
```

But I don't know if we really need the `-ish` suffixes… 🤔 🤷‍♂️

Reply:
that probably comes from the mix blend mode :P :D

Reply:
I don't think colors matter here in the docs. We don't wanna keep a track of styles inside the text documentations, unless really needed for super good reasons. If we decide to change the colors, the docs will mismatch, because we will certainly forget to update them.